Dockerfile

```diff
@@ -6,7 +6,8 @@ WORKDIR /app
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
-RUN CGO_ENABLED=0 GOOS=linux go build -o powd cmd/powd/main.go 
+RUN go build -modfile .bingo/govvv.mod -o=. github.com/ahmetb/govvv
```

Comment:
Just to keep things simple here and try another approach that bingo supports, I'm not installing bingo, and instead I'm just building govvv directly using the bingo module for it.

Reply:
Sounds good!